scripts/zmupdate.pl.in

```diff
@@ -359,12 +361,12 @@ if ( $interactive ) {
 		if ( $response =~ /^[yY]$/ ) {
 			$dbh->do(q|SET sql_mode='traditional'|); # Elevate warnings to errors
 			print "\nConverting MyISAM tables to InnoDB. Please wait.\n";
+      my $sql = "ALTER TABLE $_ ENGINE = InnoDB";
+      my $sth = $dbh->prepare_cached( $sql ) or die( "Can't prepare '$sql': ".$dbh->errstr() );
```

Comment:
Not sure this is right. These two lines go moved outside of the foreach loop just below, which means the default variable $_ won't contain the right value(s).